### PR TITLE
Distant examination of hydroponics trays

### LINKED ISSUE #3102: Examining soil/hydro trays does not tell you what plant is inside them
Would be incredibly useful for checking shit long range or as a ghost. Only way to check is to directly go up to it and click it.

### PR DESCRIPTION
**Pull request by Kurfursten**
Copies attackhand features to examine

Fixes #3102

### CODE REVIEW

**Comment by ComicIronic** [comment]
I think examine still has to inherit the parent before a new block of code.
Also, such a large chunk of code should never be copied. Make a proc with a mob/user argument, and use that to give all the messages, then put the proc in attack_hand and examine.
Also, you should not name a var usr, since that should be Byond reserved. mob/user is better, and fits a universal standard.

**Comment by clusterfack** [comment]
Adding in color macros, pfff hahaha

File: code/modules/hydroponics/hydro_tray.dm [hunk 1]
```diff
@@ -808,6 +798,28 @@
 
 			usr << "The tray's sensor suite is reporting a light level of [light_available] lumens and a temperature of [environment.temperature]K."
 
+/obj/machinery/portable/atmospherics/hydroponics/examine(mob/user)
+	..()
+	viewcontents(user)
+
+/obj/machinery/portable/atmospherics/hydroponics/proc/view_contents(mob/user)
```
Comment (ComicIronic) [inline code/modules/hydroponics/hydro_tray.dm]: portable_atmospherics

File: code/modules/hydroponics/hydro_tray.dm [hunk 2]
```diff
@@ -808,6 +798,28 @@
 
 			usr << "The tray's sensor suite is reporting a light level of [light_available] lumens and a temperature of [environment.temperature]K."
 
+/obj/machinery/portable/atmospherics/hydroponics/examine(mob/user)
```
Comment (ComicIronic) [inline code/modules/hydroponics/hydro_tray.dm]: portable_atmospherics

**Comment by ComicIronic** [comment]
Looks good.

**Pull request merged**